sdk/communication/azure-communication-callautomation/src/test/java/com/azure/communication/callautomation/EventHandlerUnitTests.java

```diff
@@ -188,7 +213,7 @@ public void parseRecognizeFailedEvent() {
             + "\"time\": \"2022-08-12T03:13:25.0252763+00:00\",\n"
             + "\"specversion\": \"1.0\",\n"
             + "\"datacontenttype\": \"application/json\",\n"
-            + "\"subject\": \"calling/callConnections/callConnectionId/RecognizeCompleted\"\n"
+            + "\"subject\": \"calling/callConnections/callConnectionId/RecognizeFailed\"\n"
```

Comment:
@cochi2 Just to confirm is this correct? Since we dont have any prefix in subject line